node/src/main/scala/org/bitcoins/node/util/BitcoinSNodeUtil.scala

```diff
@@ -1,13 +1,16 @@
 package org.bitcoins.node.util
 
+import scala.util.Random
+
 object BitcoinSNodeUtil {
 
   /** Creates a unique actor name for a actor
     * @param className
     * @return
     */
   def createActorName(className: String): String = {
-    s"${className}-${System.currentTimeMillis()}"
+    // add random number in case we gen actors at the same time
```

Comment:
I think people generally solve this problem using `AtomicInteger`

Reply:
Like adding one to `BitcoinSNodeUtil` and incrementing it?

Reply:
yup, as a `private val`